src/zimcheck.cpp

```diff
@@ -523,7 +523,7 @@ void test_articles(const zim::File& f, ErrorLogger& reporter, ProgressBar progre
                     if (s1 != s2 )
                         continue;
 
-                    reporter.setTestResult(TestType::REDUNDANT, false);
+                    reporter.setTestResult(TestType::REDUNDANT, true);
```

Comment:
Just removing the line is not an option?

Reply:
Indeed the test is fails as Matthieu pointed.